src/main/kotlin/com/sourcegraph/cody/edit/exception/EditCreationException.kt

```diff
@@ -0,0 +1,7 @@
+package com.sourcegraph.cody.edit.exception
+
+import com.sourcegraph.cody.agent.protocol.TextEdit
+
+class EditCreationException(val edit: TextEdit, cause: Throwable) : RuntimeException(cause) {
+  override val message: String = "Edit creation failed: $edit}"
```

Comment:
Looks like an extra closing brace.
```suggestion
  override val message: String = "Edit creation failed: $edit"
```